app/test/frontend/handlers/report_test.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:_pub_shared/data/account_api.dart';
+import 'package:pub_dev/fake/backend/fake_auth_provider.dart';
+import 'package:pub_dev/fake/backend/fake_email_sender.dart';
+import 'package:pub_dev/frontend/handlers/experimental.dart';
+import 'package:pub_dev/frontend/request_context.dart';
+import 'package:test/test.dart';
+
+import '../../shared/handlers_test_utils.dart';
+import '../../shared/test_services.dart';
+import '_utils.dart';
+
+void main() {
+  group('Report handlers test', () {
+    testWithProfile('requires authentication', fn: () async {
```

Comment:
We don't require that

Reply:
I'd leave it here for now. Once we allow the non-authenticated page, we can update the test.